change/@azure-msal-angular-f9ab7eec-c875-4706-8067-41fe6d7098db.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
```suggestion
  "type": "none",
```

Reply:
Force of habit, sorry. Thanks for catching it.